pkg/api/v1beta3/types_swagger_doc.go

```diff
@@ -0,0 +1,1019 @@
+//
+// TODOs are ignored from the parser. (e.g. TODO(andronat):... || TODO:...) iff
+// are on one line! For multiple line or blocks that you want to ignore use ---.
+// Any context after a --- is ignored.
+//
+// Those methods can be generated by using hack/update-generated-swagger-docs.sh
+
+// AUTO-GENERATED FUNCTIONS START HERE
+func (AWSElasticBlockStoreVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "AWSElasticBlockStoreVolumeSource represents a Persistent Disk resource in AWS.\n\nAn AWS PD must exist and be formatted before mounting to a container. The disk must also be in the same AWS zone as the kubelet. A AWS PD can only be mounted on a single machine.",
+		"volumeID":  "Unique id of the PD resource. Used to identify the disk in AWS",
+		"fsType":    "Required: Filesystem type to mount. Must be a filesystem type supported by the host operating system. Ex. \"ext4\", \"xfs\", \"ntfs\"",
+		"partition": "Optional: Partition on the disk to mount. If omitted, kubelet will attempt to mount the device name. Ex. For /dev/sda1, this field is \"1\", for /dev/sda, this field 0 or empty.",
+		"readOnly":  "Optional: Defaults to false (read/write). ReadOnly here will force the ReadOnly setting in VolumeMounts.",
+	}
+}
+
+func (Binding) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "Binding ties one object to another - for example, a pod is bound to a node by a scheduler.",
+		"metadata": "ObjectMeta describes the object that is being bound.",
+		"target":   "Target is the object to bind to.",
+	}
+}
+
+func (Capabilities) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":     "Capabilities represent POSIX capabilities that can be added or removed to a running container.",
+		"add":  "Added capabilities",
+		"drop": "Removed capabilities",
+	}
+}
+
+func (Capability) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "Capability represent POSIX capabilities type",
+	}
+}
+
+func (CauseType) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "CauseType is a machine readable value providing more detail about what occured in a status response. An operation may have multiple causes for a status (whether Failure or Success).",
+	}
+}
+
+func (ComponentConditionType) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "Type and constants for component health validation.",
+	}
+}
+
+func (ComponentStatus) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "ComponentStatus (and ComponentStatusList) holds the cluster validation info.",
+	}
+}
+
+func (Container) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                       "Container represents a single container that is expected to be run on the host.",
+		"name":                   "Required: This must be a DNS_LABEL.  Each container in a pod must have a unique name.",
```

Comment:
This map is only used from go-restful to **only** get the description, **not** order, **not** enumeration.

`go-restful` is taking the order of the fields from `reflect`. So we don't need it here.

BTW, I discussed with @caesarxuchao that we want to somehow enforce a show up order on structs to create groups. Fields it is ok, but keeping an order on the structs is going to be tricky, I have no news yet on that.